openfisca_us/parameters/gov/states/mo/tax/income/rates.yaml

```diff
@@ -0,0 +1,68 @@
+description: The tax rates for individual residents in Missouri. Thresholds are <=.
+brackets:
+- rate:
+    2021-01-01:
+      value: 0.015  
+  threshold:
+    2021-01-10:
+      value: 0
+- rate:
+    2021-01-10:
+      value: 0.02  
+  threshold:
+    2021-01-10:
+      value: 1000
+- rate:
+    2021-01-10:
+      value: 0.025
+  threshold:
+    2021-01-10:
+      value: 2000
+- rate:
+    2021-01-10:
+      value: 0.03
+  threshold:
+    2021-01-10:
+      value: 3000
+- rate:
+    2021-01-10:
+      value: 0.035
+  threshold:
+    2021-01-10:
+      value: 4000
+- rate:
+    2021-01-10:
+      value: 0.04
+  threshold:
+    2021-01-10:
+      value: 5000
+- rate:
+    2021-01-10:
+      value: 0.045
+  threshold:
+    2021-01-10:
+      value: 6000
+- rate:
+    2021-01-10:
+      value: 0.05
+  threshold:
+    2021-01-10:
+      value: 7000
+- rate:
+    2021-01-10:
+      value: 0.055
+  threshold:
+    2021-01-10:
+      value: 8000
+- rate:
+    2021-01-10:
+      value: 0.06
+  threshold:
+    2021-01-10:
+      value: 9000
+metadata:
+  reference:
+    - title: "*143.011.  Resident individuals — tax rates."
+      href: https://revisor.mo.gov/main/OneSection.aspx?section=143.011&bid=35366&hl=
+  unit: /1
+  period: year
```

Comment:
Could we add a name and label? See other MA parameters for examples

Reply:
Also, per https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#creating-scales:
```
  type: marginal_rate
  threshold_unit: currency-USD
  rate_unit: /1
```

Reply:
got it, thanks!